src/popsift/s_orientation.cu

```diff
@@ -28,6 +28,11 @@
 using namespace popsift;
 using namespace std;
 
+/* Bilinear histogram seems to be VLFeat's default now.
+ * We emulate it because we have different orientations without it.
+ */
+#define WITH_BILINEAR_ORI
```

Comment:
same comment as `WITH_VLFEAT_SMOOTHING`

Reply:
I'm just comparing the effect of every little change on the descriptors. I think that the difference is big enough to make WITH_BILINEAR_ORI a Config parameter instead of a define.